tests/drivers/virtualization/ivshmem/plain/boards/qemu_cortex_a53.conf

```diff
@@ -0,0 +1,13 @@
+CONFIG_PCIE_CONTROLLER=y
+CONFIG_PCIE_ECAM=y
+
+# QEMU PCI requires at least 256M of virtual space
+CONFIG_KERNEL_VM_SIZE=0x80000000
+
+# QEMU PCI requires physical addresses with more than 32 bits
+CONFIG_ARM64_VA_BITS_40=y
+CONFIG_ARM64_PA_BITS_40=y
+
+# CONFIG_PCIE_MSI is not set
+# CONFIG_PCIE_MSI_X is not set
+# CONFIG_PCIE_MSI_MULTI_VECTOR is not set
```

Comment:
can we remove these?

Reply:
Hello Carlo. 
Yes, we can, but these leads to other dependencies. I believe we should keep this sample like this (i.e., no MSI for ARM64) and enable MSI for a future -doorbell sample we can provide.

If we enable MSI here, we have a dependency for ITS. Then, ITS depends on two other things: first, we need to increase the IRQ size due to LPI support. We also need to provide dynamic memory, because ITS creates tables at runtime (it asked the hardware for for baser page size).

Let's say we enable all these options; this sample (ivshmem-plain) does not use MSI at all, and will not work if you enable ivshmem-doorbell (it requires that `ivshmem_get_vectors` returns zero). This is the intended scenario for ivshmem-plain: just shared memory.

Our proposal is: provide another sample for ivshmem-doorbell and put a comment back in this file saying something like "if you really want to enable MSI here, look at ivshmem-doorbell sample for dependencies related to MSI"

Let me know if you agree or how do you prefer we proceed.

Reply:
Sorry, probably I was not clear enough. I was not suggesting to enable those but to simply remove those from the conf file entirely since they are commented out. Or if you want to keep those commented out, add a comment explaining why.

Reply:
oooh, yes. I did not get that.
I rewrote history with rebase.. I'm not sure that is the preferred way to address this? If you can, let me know for future reference.

new/changed commits:
https://github.com/zephyrproject-rtos/zephyr/pull/52823/commits/95d55f1d239a69323210b06a7d248e9ef664e328: new commit, protected inclusion of `msi.h` header, so that we can disable MSI for this sample

https://github.com/zephyrproject-rtos/zephyr/pull/52823/commits/cd22952660223b1d233ff9f0717e5396c2b60255: new commit. Remove MSI configuration options from the sample.

https://github.com/zephyrproject-rtos/zephyr/pull/52823/commits/028f88b480a7ae732f6a65a95dbc3f1be35972e3: changed to remove the special `#` lines for `qemu_cortex_a53`

https://github.com/zephyrproject-rtos/zephyr/pull/52823/commits/fe6173fb3e99a4cf2fc69a2c0fda4430118c69fc: changed to remove the special `#` lines for `qemu_kvm_arm64`